### PR TITLE
feature/add-yanzi

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/yanzi/comfort/uplink.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+function consume(event) {
+  const { resourceType } = event.data.list[0];
```
Comment (mariusgiger) [inline types/yanzi/comfort/uplink.js:2]: emit a sample for each of the values

File: types/yanzi/motionPlus/uplink.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+function consume(event) {
+  const { resourceType } = event.data.list[0];
```
Comment (mariusgiger) [inline types/yanzi/motionPlus/uplink.js:2]: emit a sample for all items in the list

**Review by mariusgiger — CHANGES_REQUESTED** [review]
@giepetto please align the format with @J2thatsme

File: types/yanzi/comfort/uplink.js (hunk 1 above)
Comment (J2thatsme) [reply types/yanzi/comfort/uplink.js:2]: I never saw more than one value in the "list". But i might as well loop over it

File: types/yanzi/comfort/uplink.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+function consume(event) {
+  let topic = "default";
+
+  event.data.list.forEach((element) => {
```
Comment (mariusgiger) [inline types/yanzi/comfort/uplink.js:4]: please change this to the format that is used for array uplinks as described by Geppetto

Comment (J2thatsme) [reply types/yanzi/comfort/uplink.js]: @mariusgiger done

**Review by mariusgiger — APPROVED** [review]

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**